.circleci/config.yml

```diff
@@ -671,7 +683,6 @@ workflows:
           component: flowmachine
           requires:
             - run_flowmachine_tests
-            - integration_tests
```

Comment:
Just to check: am I correct to assume that these can be removed because they are implicit in the re-defined workflow?

Reply:
Yeah, I think the critical bit really is that unless _everything_ works, the build will never get tagged as latest, nor receive a release tag. Doesn't matter so much if the commit-hash tagged image gets pushed up to docker cloud in my view.